src/utils/calculator.test.ts

```diff
@@ -0,0 +1,36 @@
+import { GatewayOracle } from 'ardrive-core-js';
+import type { ArweaveOracle } from 'ardrive-core-js/lib/arweave_oracle';
+import { expect } from 'chai';
+import { SinonSpiedInstance, spy } from 'sinon';
+import { Calculator } from './calculator';
+
+const someValidByteCount = 1000; // 1KB
+
+describe('Ar<>Data calculator', () => {
+	let spyedOracle: SinonSpiedInstance<ArweaveOracle>;
+	let calculator: Calculator;
+
+	before(() => {
+		spyedOracle = spy(new GatewayOracle());
+		calculator = new Calculator(spyedOracle);
+	});
+
+	it('Zero oracle calls when just constructed', () => {
+		expect(spyedOracle.getWinstonPriceForByteCount.callCount === 0);
+	});
+
+	it('Undefined estimation if not setted up', () => {
```

Comment:
Nit: everywhere you say `setted` say `set` instead (it's the past tense of the verb set :) )

Reply:
Fixed this